elabapy/baseapi.py

```diff
@@ -25,6 +25,7 @@ class BaseAPI(object):
 
     token: str = ""
     endpoint: str = ""
+    proxies: dict = None
```

Comment:
So you can't have a dict with a default value of None. It has to be `{}`. Also you should be more specific: `Dict[str, str]`

Reply:
Ok for {} :-)
I choosed a default value of None because I thought I saw that was the default value in the **kwargs of the Requests module.
I'm trying to find a time tomorrow to do some live testing to see if a {} value is ok.

Reply:
Then Optional might be a better choice: https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type

Reply:
Great, `Optional ` seems a good solution :-)  ...and it now looks functionally like real python ;-)
Works fine for me with and without proxies.
```
manager = elabapy.Manager(endpoint=endpoint, token=token, proxies=proxies)
manager = elabapy.Manager(endpoint=endpoint, token=token)
```